components/DepartmentsWalkthrough.vue

```diff
@@ -0,0 +1,297 @@
+          budget: "$x mil",
+          display: "none",
+          sliderMax: 16452,
+          storedValue: 0
+        },
+        {
+          department: "general-city-responsibilities",
+          header: "General City Responsibilities",
+          content: "",
+          budget: "$x mil",
+          display: "none",
+          sliderMax: 16452,
+          storedValue: 0
+        },
+        {
+          department: "human-welfare-neighborhood-development",
+          header: "Human Welfare & Neighborhood Development",
+          content: "",
+          budget: "$x mil",
+          display: "none",
+          sliderMax: 16452,
+          storedValue: 0
+        },
+        {
+          department: "public-protection",
+          header: "Public Protection",
+          content: "",
+          budget: "$x mil",
+          display: "none",
+          sliderMax: 16452,
+          storedValue: 0
+        },
+        {
+          department: "public-works-transportation-commerce",
+          header: "Public Works, Transportation, & Commerce",
+          content: "",
+          budget: "$x mil",
+          display: "none",
+          sliderMax: 16452,
+          storedValue: 0
+        }
+      ],
+      buttons: [
+        {
+          name: "prev",
+          text: "prev",
+          disabled: false,
+          outlined: true,
+          method: "previous",
+          class: "",
+          render: false
+        },
+        {
+          name: "next",
+          text: "next",
+          disabled: false,
+          outlined: false,
+          method: "next",
+          class: "white--text",
+          render: false
+        },
+        {
+          name: "next-disabled",
+          text: "next",
+          disabled: true,
+          outlined: false,
+          method: "next",
+          class: "",
+          render: true
+        }
+      ]
+    };
+  },
+  computed: {
+    buttonsShow: function() {
+      return this.buttons.filter(function(button) {
+        return button.render;
+      });
+    }
+  },
+  methods: {
+    expand() {
+      const Text = document.querySelector(".Category-Content-Text");
+      Text.style.overflow = "inherit";
+      var Btn = document.getElementById("btn");
+      Btn.remove();
+    },
+    tradeBtns() {
+      this.buttons[1].render = true;
+      this.buttons[2].render = false;
+    },
+    navigate(button) {
```

Comment:
I eventually got an index out of bounds error when progressing past the last department, not sure exactly where the issue is
`TypeError: this.departments[this.index] is undefined`

Reply:
i found the source in the CSS and removed it, the last slide was set to opacity of 0. i also added a "finish" button on the last slide that so far has no function